<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad - Untitled[*]</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">.QTextEdit {
	background-color: rgb(50, 50, 50);
	color: rgb(250, 250, 250);
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actPrint"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actUndo"/>
    <addaction name="actRedo"/>
    <addaction name="separator"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="actCut"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actAbout"/>
    <addaction name="actAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="separator"/>
   <addaction name="actPrint"/>
   <addaction name="separator"/>
   <addaction name="actUndo"/>
   <addaction name="actRedo"/>
   <addaction name="separator"/>
   <addaction name="actCopy"/>
   <addaction name="actCut"/>
   <addaction name="actPaste"/>
  </widget>
  <action name="actNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/document.png</normaloff>:/pix/images/icons/16/document.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/folder-open-document.png</normaloff>:/pix/images/icons/16/folder-open-document.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/disk--pencil.png</normaloff>:/pix/images/icons/16/disk--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/printer.png</normaloff>:/pix/images/icons/16/printer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/document-copy.png</normaloff>:/pix/images/icons/16/document-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/clipboard-paste.png</normaloff>:/pix/images/icons/16/clipboard-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/arrow-left.png</normaloff>:/pix/images/icons/16/arrow-left.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/arrow-right.png</normaloff>:/pix/images/icons/16/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y, Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/scissors-blue.png</normaloff>:/pix/images/icons/16/scissors-blue.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>&amp;About NotePad</string>
   </property>
  </action>
  <action name="actSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pix/images/icons/16/disk.png</normaloff>:/pix/images/icons/16/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actPaste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actUndo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actRedo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>actRedo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>actUndo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actCopy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actCut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
